po/de/minetest.po

```diff
@@ -1278,7 +1278,7 @@ msgid "Camera update enabled"
 msgstr "Kameraaktualisierung aktiviert"
 
 #: src/client/game.cpp
-msgid "Can't show block bounds (need 'basic_debug' privilege)"
+msgid "Can't show block bounds (disabled by mod or game)"
 msgstr ""
 "Blockgrenzen können nicht gezeigt werden („basic_debug“-Privileg benötigt)"
```

Comment:
Line remains

Reply:
These files are updated by the translation update script, and shouldn't be modified in normal PRs